src/Features/Core/Portable/CodeRefactorings/AbstractRefactoringHelpersService.cs

```diff
@@ -346,7 +346,7 @@ protected virtual IEnumerable<SyntaxNode> ExtractNodesSimple(SyntaxNode? node, I
             // `var a = b;` | `var a = b`;
             if (syntaxFacts.IsLocalDeclarationStatement(node) || syntaxFacts.IsLocalDeclarationStatement(node.Parent))
             {
-                var localDeclarationStatement = syntaxFacts.IsLocalDeclarationStatement(node) ? node : node.Parent;
+                var localDeclarationStatement = syntaxFacts.IsLocalDeclarationStatement(node) ? node : node.Parent!;
```

Comment:
there is a GetRequiredParent() extension that could beused here.

Reply:
This is a case where we shouldn't be needing the `GetRequiredParent()` method (the use of a runtime assertion is undesirable because static code flow _should_ be able to handle the case).